index.html

```diff
@@ -0,0 +1,215 @@
+    <title>Zadanie 5</title>
+    <link rel="stylesheet" href="./styles/base.css">
+    <link rel="stylesheet" href="./styles/header.css">
+    <link rel="stylesheet" href="./styles/content.css">
+    <link rel="stylesheet" href="./styles/footer.css">
+</head>
+
+<body>
+
+    <!-- Pytanie czy w VSC moge tworzyć własne szablony i donieśc się do nich jak w Emmet? -->
+
+    <!-- Header start -->
+
+    <header class="header row">
+        <div class="header__container container">
+            <h1 class="header__logo cell">
+                <a href="#"><img src="images/logo.svg" alt=""></a>
+            </h1>
+            <nav class="header__nav cell">
+                <ul>
+                    <li><a href="#">Team</a></li>
+                    <li><a href="#">Pricing</a></li>
+                    <li><a href="#">Features</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <!-- Slider -->
+
+    <section data-id="Slider" class="content row">
+        <div class="content__container container">
+            <article class="content__article cell left">
+                <title>Beautiful Free Nova template</title>
+                <p>A top notch premium quality template for your next web project.</p>
+                <button type="button">Download FREE!</button>
+            </article>
+            <article class="content__article cell right">
+                <a href="#"><img class="screen" src="images/screen.png" alt=""></a>
+            </article>
+        </div>
+    </section>
+
+    <!-- Features -->
+
+    <section data-id="Features" class="content row">
+        <div class="content__container container">
+            <article class="content__article title--module">FEATURES</article>
+        </div>
+        <div class="content__container container">
+            <article class="content__article cell">
+                <div>
+                    <img src="images/responsive.svg" alt="">
+                </div>
+                <title>fully responsive</title>
+                <p>Look amazing on any device: smartphone, tablet, laptop and desktop</p>
+            </article>
+            <article class="content__article cell">
+                <div>
+                    <img src="images/customizable.svg" alt="">
+                </div>
+                <title>customizable</title>
+                <p>Look amazing on any device: smartphone, tablet, laptop and desktop</p>
+            </article>
+            <article class="content__article cell">
+                <div>
+                    <img src="images/design.svg" alt="">
+                </div>
+                <title>slick and beautiful design</title>
+                <p>Look amazing on any device: smartphone, tablet, laptop and desktop</p>
+            </article>
+        </div>
+    </section>
+
+    <!-- Module cooperating companies -->
+
+    <section data-id="Module cooperating" class="content row">
+
+        <article class="content__article cell">
```

Comment:
Zdecydowanie to nie `article` - mam wrażenie, że jak kiedyś się nadużywało `div` to ty teraz to wszystkiego starasz się użyć `article`. TO nie jest dobre rozwiązanie. Tutaj zdecydowanie lepioej będzie pasować `aside`.